src/DI/TranslationExtension.php

```diff
@@ -219,7 +219,7 @@ public function beforeCompile(): void
 			if ($this->config->logger === true) {
 				$psrLogger = $builder->getDefinitionByType(Psr\Log\LoggerInterface::class);
 
-			} elseif (is_string($this->config->logger)) {
+			} elseif (is_string($this->config->logger) && class_exists($this->config->logger)) {
```

Comment:
PHPStan fails on this row? Because I think, that Exception from **ReflectionClass** is better.

Reply:
yeah, you need to pass class-string (which is sth checked via class_exists) to reflection class constructor.